internal/interfaces/grpc/handler/account.go

```diff
@@ -141,39 +140,55 @@ func (a *account) Balance(
 func (a *account) ListUtxos(
 	ctx context.Context, req *pb.ListUtxosRequest,
 ) (*pb.ListUtxosResponse, error) {
-	name, err := parseAccountName(req.GetAccountName())
+	namespace, err := parseAccountNamespace(req.GetNamespace())
 	if err != nil {
 		return nil, status.Error(codes.InvalidArgument, err.Error())
 	}
 
-	utxosInfo, err := a.appSvc.ListUtxosForAccount(ctx, name)
+	utxosInfo, err := a.appSvc.ListUtxosForAccount(ctx, namespace)
 	if err != nil {
 		return nil, err
 	}
 	spendableUtxos := parseUtxos(utxosInfo.Spendable.Info())
 	lockedUtxos := parseUtxos(utxosInfo.Locked.Info())
 	return &pb.ListUtxosResponse{
 		SpendableUtxos: &pb.Utxos{
-			AccountName: name,
-			Utxos:       spendableUtxos,
+			Namespace: namespace,
+			Utxos:     spendableUtxos,
 		},
 		LockedUtxos: &pb.Utxos{
-			AccountName: name,
-			Utxos:       lockedUtxos,
+			Namespace: namespace,
+			Utxos:     lockedUtxos,
 		},
 	}, nil
 }
 
 func (a *account) DeleteAccount(
 	ctx context.Context, req *pb.DeleteAccountRequest,
 ) (*pb.DeleteAccountResponse, error) {
-	name, err := parseAccountName(req.GetAccountName())
+	namespace, err := parseAccountNamespace(req.GetNamespace())
 	if err != nil {
 		return nil, status.Error(codes.InvalidArgument, err.Error())
 	}
 
-	if err := a.appSvc.DeleteAccount(ctx, name); err != nil {
+	if err := a.appSvc.DeleteAccount(ctx, namespace); err != nil {
 		return nil, err
 	}
 	return &pb.DeleteAccountResponse{}, nil
 }
+
+func (a *account) SetAccountLabel(
+	ctx context.Context,
+	req *pb.SetAccountLabelRequest,
+) (*pb.SetAccountLabelResponse, error) {
+	namespace, err := parseAccountNamespace(req.GetNamespace())
+	if err != nil {
+		return nil, status.Error(codes.InvalidArgument, err.Error())
+	}
```

Comment:
According to prev required changes, this should be (note that we are currently missing to validate the label):
```suggestion
	namespace, err := parseAccountName(req.GetNamespace())
	if err != nil {
		return nil, status.Error(codes.InvalidArgument, err.Error())
	}
	label, err := parseAccountName(req.GetLabel())
	if err != nil {
		return nil, status.Error(codes.InvalidArgument, err.Error())
	}
```

Reply:
since `parseAccountName()` makes sure that the given string is not empty, we can re-use it for both namespace and label. This is the only check we have to make at interface layer to validate request arguments. The app layer will tell if they are valid or not after using them eventually.